spec/index.bs

```diff
@@ -7575,31 +7575,39 @@ GPUQueue includes GPUObjectBase;
-                - |copySize|.[=Extent3D/depthOrArrayLayers=] must be `1`.
-                - |textureDesc|.{{GPUTextureDescriptor/usage}} must include both {{GPUTextureUsage/RENDER_ATTACHMENT}} and {{GPUTextureUsage/COPY_DST}}.
-                - |textureDesc|.{{GPUTextureDescriptor/dimension}} must be {{GPUTextureDimension/"2d"}}.
-                - |textureDesc|.{{GPUTextureDescriptor/format}} must be one of the following:
-                    - {{GPUTextureFormat/"rgba8unorm"}}
-                    - {{GPUTextureFormat/"rgba8unorm-srgb"}}
-                    - {{GPUTextureFormat/"bgra8unorm"}}
-                    - {{GPUTextureFormat/"bgra8unorm-srgb"}}
-                    - {{GPUTextureFormat/"rgb10a2unorm"}}
-                    - {{GPUTextureFormat/"rgba16float"}}
-                    - {{GPUTextureFormat/"rgba32float"}}
-                    - {{GPUTextureFormat/"rg8unorm"}}
-                    - {{GPUTextureFormat/"rg16float"}}
-            </div>
-
-            Issue: The above list was written just for ImageBitmap.
-            Re-evaluate it for canvas (and video).
-            (It's probably still fine, though `rgb10a2unorm` is a bit of an outlier.)
+            1. If |sourceImage| is an {{ImageBitmap}} and
+                |sourceImage|.{{platform object/[[Detached]]}} is true,
+                throw an {{InvalidStateError}} and stop.
+            1. If any of the following requirements are unmet, throw an {{OperationError}} and stop.
+                <div class=validusage>
+                    - |copySize|.[=Extent3D/depthOrArrayLayers=] must be `1`.
+                </div>
+            1. Issue the following steps on the [=Device timeline=] of |this|:
+                <div class=device-timeline>
+                    1. Let |textureDesc| be |destination|.{{GPUImageCopyTexture/texture}}.{{GPUTexture/[[descriptor]]}}.
+                    1. If any of the following requirements are unmet, generate a validation error and stop.
+                        <div class=validusage>
+                            - |textureDesc|.{{GPUTextureDescriptor/usage}} must include both
+                                {{GPUTextureUsage/RENDER_ATTACHMENT}} and {{GPUTextureUsage/COPY_DST}}.
+                            - |textureDesc|.{{GPUTextureDescriptor/dimension}} must be {{GPUTextureDimension/"2d"}}.
+                            - |textureDesc|.{{GPUTextureDescriptor/format}} must be one of the following:
+                                - {{GPUTextureFormat/"rgba8unorm"}}
+                                - {{GPUTextureFormat/"rgba8unorm-srgb"}}
+                                - {{GPUTextureFormat/"bgra8unorm"}}
+                                - {{GPUTextureFormat/"bgra8unorm-srgb"}}
+                                - {{GPUTextureFormat/"rgb10a2unorm"}}
+                                - {{GPUTextureFormat/"rgba16float"}}
+                                - {{GPUTextureFormat/"rgba32float"}}
```

Comment:
I remember in this comment, it seems that we won't support float format https://chromium-review.googlesource.com/c/chromium/src/+/2862407/15..18/third_party/blink/renderer/modules/webgpu/gpu_queue.cc#b456, is it still true?

Reply:
Oops, yes, I intended to do that at the same time I updated the issue just below here, but forgot. Thank you for reminding me!